### ISSUE TITLE
[Bug]: [vite-node] [ERR_LOAD_URL] vuetify/styles

### ISSUE THREAD

**Issue by thejacer87** [invictus-codes/nuxt-vuetify#20]
### What happened?

I installed the plugin per the instructions with pnpm: `pnpm add -D @invictus.codes/nuxt-vuetify`

When I start the Nuxt server I see this error in the console:

```
[10:41:01 p.m.]  ERROR  Failed to resolve import "vuetify/styles" from "virtual:nuxt:/path/to/my/project/.nuxt/css.mjs". Does the file exist?

[10:41:01 p.m.]  ERROR  Failed to resolve import "vuetify" from "virtual:nuxt:/path/to/my/project/.nuxt/templates.plugin.12f8b498.mts". Does the file exist?
```

And when I try to open any page, I see this:
![image](https://github.com/invictus-codes/nuxt-vuetify/assets/6412919/8d20e1bb-e742-4b0a-999d-f60d6614c1e1)

### Version

1.0.2 (Default)

### What browsers are you seeing the problem on?

Firefox, Chrome

### Relevant log output

```shell
[10:41:01 p.m.]  ERROR  Failed to resolve import "vuetify/styles" from "virtual:nuxt:/path/to/my/project/.nuxt/css.mjs". Does the file exist?

[10:41:01 p.m.]  ERROR  Failed to resolve import "vuetify" from "virtual:nuxt:/path/to/my/project/.nuxt/templates.plugin.12f8b498.mts". Does the file exist?
```

**Comment by thejacer87** [comment]
so it seems like my issue is related to something going on with nuxt and pnpm workspaces: https://github.com/nuxt/nuxt/issues/14146

i installed it from the root of the monorepo with `pnpm -F myproj add -D @invictus.codes/nuxt-vuetify` 

if i install it directly in the project folder, it worked as expected

**Comment by gwenf** [comment]
I just wanted to add a comment here in case someone else runs into this issue.
I was getting this error from `npm run dev` on a new Nuxt project after I added this nuxt-vuetify module.

Steps I used to resolve it:
1. Made sure npm was up to date (currently working on Node `18.17.1` and npm `9.6.7`)
2. Deleted `node_modules` folder and the package-lock.json file
3. Ran `npm i` again
4. `npm run dev` now works
